pkg/js/util/exceptions.ts

```diff
@@ -84,18 +70,14 @@ const createTupleUsersetRequireDirectError = (props: BaseProps) => {
     }));
 };
 
-const createNoEntryPointError = (props: BaseProps, validRelations: string) => {
-    const data = getValidRelationsArray(validRelations);
+const createNoEntryPointError = (props: BaseProps) => {
     const { errors, lines, lineIndex, symbol } = props;
-    const isInvalid = !data?.includes(symbol);
-    if (isInvalid) {
-        errors.push(constructValidationError({
-            message: `\`${symbol}\` is an impossible relation (no entrypoint).`,
-            lines,
-            lineIndex,
-            metadata: { symbol, errorType: ValidationError.RelationNoEntrypoint, relation: symbol },
-        }));
-    }
+    errors.push(constructValidationError({
+        message: `\`${symbol}\` is an impossible relation (no entrypoint).`,
+        lines,
+        lineIndex,
+        metadata: { symbol, errorType: ValidationError.RelationNoEntrypoint, relation: symbol },
```

Comment:
Maybe we should add the type here so it's clearer to users?

Reply:
`${symbol}` is an impossible relation for `${typeName}` (no entrypoint). ?